README.md

```diff
@@ -4,6 +4,14 @@
 * Add AKS clusters to kubeconfig via the GUI
 * Run AppLens detector diagnostics on your AKS clusters
 
+## Telemetry
+
+This extension collects telemetry data to help us build a better experience for building applications with Kubernetes and VS Code. We only collect the following data:
```

Comment:
Consider "Azure Kubernetes Service" rather than just "Kubernetes" since this extension is specific to AKS.  (The existing wording still works fine though, if you prefer it.)